config/sus.rb

```diff
@@ -5,3 +5,4 @@
 
 require 'covered/sus'
 include Covered::Sus
+FileUtils.mkdir_p(::File.expand_path(ENV['XDG_STATE_HOME'] || "~/.local/state"))
```

Comment:
```suggestion

require 'fileutils'
FileUtils.mkdir_p(::File.expand_path(ENV['XDG_STATE_HOME'] || "~/.local/state"))
```

Reply:
I'm surprised that it works on my machine (running 3.3.0) - something in my environment must be causing FileUtils to be loaded. Hrm.